File: java_backup/java.jimut/p_prog/HCF.java
import java.io.*;
class HCF
{
public static void main(String[]args)
{
int a=4,b=6,c=8,d=5,k;
if(a>b)
{
c=a;
d=b;
}
else
{
c=b;
d=a;
}
while(c%d!=0)
{
k=c%d;
c=d;
d=k;
}
System.out.println("HCF"+d);
}
}

File: 3rd_sem_sxc/SEM==3/jimut linux progs/hcf.pl
system("clear");
printf("\n Enter first number");
$a=<STDIN>;
printf("\n Enter 2nd number");
$b=<STDIN>;
$x=$a;
$y=$b;
$r=$x%$y;
while($r!=0)
{
	$x=$y;
	$y=$r;
	$r=$x%$y;
}
$h=$y;
$l=$a*$b/$h;
printf("hcf = %d lcm = %d of %d and %d",$h,$l,$a,$b);
#end of program
